File: test/tests/code/io_stuff.js
function ioStuff()
{
	console.log('This is stdout data.');
	console.error('This is stderr data.');
	
	let data = '';
	
	process.stdin.once('readable', () =>
	{
		let chunk;
		while (chunk = process.stdin.read()) {
			data += chunk;
		}
		
		console.log('The stdin data: ' + data);
	});
}

File: test/tests/code/io_stuff.cs
class IOStuff
{
	public static void ioStuff()
	{
		System.Console.Out.WriteLine("This is stdout data.");
		System.Console.Error.WriteLine("This is stderr data.");
		
		string data = System.Console.In.ReadLine();
		System.Console.Out.WriteLine("The stdin data: " + data);
	}
}
